src/6_branch.js

```diff
@@ -541,6 +541,39 @@ Branch.prototype['init'] = wrap(
 );
 
 /**
+ * @function Branch.deepview
+ * @param {function(?Error, utils.sessionData=)=} callback - _optional_ - callback to read the session data.
```

Comment:
we wnat the callback to accept a function/url for the CTA, right? or do we want a separate/always global function to handle a CTA click?